include/ppx/application.h

```diff
@@ -343,12 +343,12 @@ struct ApplicationSettings
         bool headless = false;
 #endif
         bool                listGpus              = false;
-        std::string         metricsFilename       = std::filesystem::current_path().string();
+        std::string         metricsFilename       = "report_@.json";
         bool                overwriteMetricsFile  = false;
         std::pair<int, int> resolution            = std::make_pair(0, 0);
         uint32_t            runTimeMs             = 0;
         int                 screenshotFrameNumber = -1;
-        std::string         screenshotPath        = "";
+        std::string         screenshotPath        = "screenshot_frame_#.ppm";
```

Comment:
not necessarily for this PR, but should we add validation on these to make sure there are not empty?

Reply:
I believe passing an empty string for the screenshot path would cause an assert ("Partial path cannot be empty string") in the code when `GetFullPath(mStandardOpts.pScreenshotPath->GetValue()...)` is called in the screenshot code. Example in unit tests: `GetFullPath_PartialPathEmpty`

Reply:
It wouldn't trigger until the screenshot frame though, I don't think this is checked during flag parsing or anything.